### PR TITLE
Replace charts.axway.com with repository.axway.com

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Due to security risks, change https://charts.axway.com to https://repository.axway.com

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | d1d9602 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-465.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by jcollins-axway — CHANGES_REQUESTED** [review]

File: content/en/docs/connect_manage_environ/connect_api_manager/deploy-your-agents-with-amplify-cli.md [hunk 1]
```diff
@@ -256,13 +256,15 @@ persistentVolumeClaimConfig:
     storageClass: gp2-csi
 ```
 
+The helm charts for the agents and download instructions can be found at https:/repository.axway.com/catalog.
 The agents can be deployed with the following commands, which are mentioned at the end of the CLI install prompts:
 
 ```bash
-helm repo add axway https://charts.axway.com/charts
+helm repo add axway https://helm.repository.axway.com --username==<client-id> --password=<client_secret>
+helm repo pull <agent_filename>
```
Comment (jcollins-axway) [inline content/en/docs/connect_manage_environ/connect_api_manager/deploy-your-agents-with-amplify-cli.md:264]: get rid of this line

**Review by jcollins-axway — APPROVED** [review]

File: content/en/docs/connect_manage_environ/connect_api_manager/deploy-your-agents-with-amplify-cli.md (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/connect_manage_environ/connect_api_manager/deploy-your-agents-with-amplify-cli.md]: removed

File: content/en/docs/connect_manage_environ/connect_api_manager/deploy-your-agents-with-amplify-cli.md [hunk 2]
```diff
@@ -259,8 +259,8 @@ persistentVolumeClaimConfig:
 The agents can be deployed with the following commands, which are mentioned at the end of the CLI install prompts:
 
 ```bash
-helm repo add axway https://charts.axway.com/charts
-helm repo update
+helm repo add axway https://repository.axway.com --username==<client-id> --password=<client_secret>
+helm repo pull <agent_filename>
 helm upgrade --install --namespace <YOUR_NAMESPACE> v7-discovery axway/v7-discovery -f da-overrides.yaml
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/connect_manage_environ/connect_api_manager/deploy-your-agents-with-amplify-cli.md]: fixed all the names and urls

**Review by robertengaxway — APPROVED** [review]
reviewed with Jason

**Pull request merged**